src/java/relex/PlainTextServer.java

```diff
@@ -258,8 +263,9 @@ else if (args[i].equals("--verbose") )
 						String fin = StanfordView.printRelations(parse, true, "    ");
 						out.println(fin);
 					}
-					if (verbose)
-						System.out.print(SimpleView.printRelations(parse));
+					if (logger.isDebugEnabled()) {
+						logger.debug("{}", SimpleView.printRelations(parse));
+					}
 				}
```

Comment:
So, here, it would make more sense to have both the old and the new code: viz, it always prints to the logger, if dubugging is enabled. However, it only prints to stdout if `--verbose` is requested.

To be clear: at this point, with relex, I kind-of don't care any more. I'm just describing the "industry standard" definition of what the `--verbose` flag does, in all other applications. Under the principle of least-surprise, relex should do exactly what everyone else does (viz, print to stdout when  user asks for verbose)

Reply:
I reverted back changes for for classes Server, ServerSession, and PlainTextServer.
Now if --verbose option is set all is printed into stdout as it was before for these classes.